scripts/gather-mocked-data.js

```diff
@@ -158,6 +158,20 @@ function gatherSecretRefs(client, serverAddr, data) {
   }, Promise.resolve(data));
 }
 
+function obfuscateKeys(data) {
+  // replace secret keys with dummy data
+  // to avoid commiting real cluster data
+  mockedData = Buffer.from('mocked-data').toString('base64')
+  let i = data.clusters._host['api/v1/namespaces/mig/secrets']
+  let secrets = Object.values(i)
+  secrets.map(value => {
+    value.data['aws-access-key-id'] = mockedData;
+    value.data['aws-secret-access-key'] = mockedData;
+    value.data['aws-secret-access-key-id'] = mockedData;
```

Comment:
Let's remove 'aws-secret-access-key-id'

Real data will not have this key.  It is in our initial mocked data because I was running older code.  New code no longer uses it, so we don't need/want to address it.

See https://github.com/fusor/mig-controller/blob/master/pkg/apis/migration/v1alpha1/migstorage_types.go#L40 for more info on the used values.